cmd/plugin.go

```diff
@@ -416,58 +429,132 @@ func runPluginUpdateCmd(cmd *cobra.Command, args []string) {
 		if image.Config.Plugin.Version != "" {
 			versionString = " v" + image.Config.Plugin.Version
 		}
-		fmt.Printf("Updated plugin: %s%s\n", constants.Bold(report.Plugin.Name), versionString)
 		org := image.Config.Plugin.Organization
 		name := image.Config.Plugin.Name
+		docURL := ""
 		if org == "turbot" {
-			fmt.Printf("Documentation:  https://hub.steampipe.io/plugins/%s/%s\n", org, name)
+			docURL = fmt.Sprintf("https://hub.steampipe.io/plugins/%s/%s", org, name)
 		}
-		// fmt.Println()
+		updateReports = append(updateReports, installReport{
+			Plugin:         fmt.Sprintf("%s@%s", report.CheckResponse.Name, report.CheckResponse.Stream),
+			Skipped:        false,
+			Version:        versionString,
+			DocURL:         docURL,
+			IsUpdateReport: true,
+		})
 	}
 
-	if len(updateSkipped) > 0 {
-		skipReasons := []string{}
-		notUpdatedSinceNotInstalled := []string{}
-		for _, s := range updateSkipped {
-			skipReasons = append(skipReasons, s.String())
-			if s.reason == "Not Installed" {
-				notUpdatedSinceNotInstalled = append(notUpdatedSinceNotInstalled, s.plugin)
+	refreshConnectionsIfNecessary(updateReports, true)
+	printInstallReports(updateReports, true)
+
+	// a concluding blank line at the bottom, since we have multiple
+	// lines in the output - no matter the outcome
+	fmt.Println()
+}
+
+// Prints out the installation reports onto the console
+func printInstallReports(reports []installReport, isUpdateReport bool) {
+	installedOrUpdated := []installReport{}
+	canBeInstalled := []installReport{}
+	canBeUpdated := []installReport{}
+
+	for _, report := range reports {
+		report.IsUpdateReport = isUpdateReport
+		if !report.Skipped {
+			installedOrUpdated = append(installedOrUpdated, report)
+		} else if report.SkipReason == "Not installed" {
```

Comment:
move ""Not installed" and ""Already installed" into consts - maybe in installreport.go